driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -54,6 +52,22 @@
 
     private static final String CASSANDRA_VERSION_REGEXP = "\\d\\.\\d\\.\\d+(-\\w+)?";
 
+    public static final String DEFAULT_CLIENT_TRUSTSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_CLIENT_TRUSTSTORE_PATH = "/client.truststore";
+
+    public static final String DEFAULT_CLIENT_KEYSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_CLIENT_KEYSTORE_PATH = "/client.keystore";
+
+    public static final String DEFAULT_SERVER_TRUSTSTORE_PASSWORD = "cassandra1sfun";
+    public static final String DEFAULT_SERVER_TRUSTSTORE_PATH = "/server.truststore";
+
+    private static final File DEFAULT_SERVER_TRUSTSTORE_FILE = createTempStore(DEFAULT_SERVER_TRUSTSTORE_PATH);
```

Comment:
Ah, makes sense. Why not use it for client stores as well then?

I'm fine keeping it this way but I would add a comment to explain.